pkg/api/types.go

```diff
@@ -140,7 +140,9 @@ type ObjectMeta struct {
 	Annotations map[string]string `json:"annotations,omitempty"`
 
 	// List of objects depended by this object. If ALL objects in the list have
-	// been deleted, this object will be garbage collected.
+	// been deleted, this object will be garbage collected. If this object is managed
+	// by a controller and OwnerReference pointing to this controller has a controller filed
+	// set to true. There can be only one managing controller.
```

Comment:
In that case (the current wording has typos and is confusing):

If this object is managed by a controller, then an entry in this list will point to this controller, with the controller field set to true. There cannot be more than one managing controller.